spec/models/payment_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Payment, type: :model do
+  it { should belong_to(:site) }
+  it { should belong_to(:agency) }
+  it { should validate_numericality_of(:amount).is_greater_than(0.00) }
+  it { should validate_numericality_of(:discrepancy) }
```

Comment:
If we add the presence validations in the model, make sure to add the specs for that here